cmr-exchange/ses-plugin/src/cmr/sizing/formats.clj

```diff
@@ -44,11 +44,18 @@
       (get-in [:umm :SizeEstimation :AverageSizeOfGranulesSampled])
       read-number))
 
+(defn- get-netcdf4-rate
+  "Gets the :Rate value for :NetCDF-4 format in avg-comp-info."
+  [avg-comp-info]
+  (some #(when (= "NetCDF-4" (:Format %)) (:Rate %)) avg-comp-info))  
```

Comment:
It's probably worth a `def` for the NetCDF-4 string.

Reply:
Done.